### DISCUSSION TITLE
ERROR: Charging status not available from api

### DISCUSSION THREAD

**Discussion by ioudeveldhuis** [flobz/psa_car_controller#366] [Q&A]
In my log file I see the following error passing by every 2 minutes:
10 13:28:59,901 :: ERROR :: charging status not available from api
10 13:30:58,123 :: ERROR :: charging status not available from api
10 13:32:58,576 :: ERROR :: charging status not available from api
10 13:34:58,239 :: ERROR :: charging status not available from api
Is this an error from the controller (incorrect command send) or is this on the PSA end (functionality not available through API for certain country's)?

**Comment by BlackyHole99** [comment]
I can observe the same behavior on my installation. Think I found the responsible line in the code (psa_car_controller/psacc/application/psa_client.py#206), but I‘m not sure if the json model provided by PSA has changed or something else is forcing the ‚AttributeError‘.

**Reply by BlackyHole99** [reply]
Short update: I did my first charging yesterday while running the psa_car_controller. After the first minutes of charging the error message disappeared. So maybe an initial value or something else is missing after a fresh install. Now the log looks fine.
